funsor/montecarlo.py

```diff
@@ -44,6 +44,7 @@ def monte_carlo_interpretation(**sample_inputs):
 
 @monte_carlo.register(Integrate, Funsor, Funsor, frozenset)
 def monte_carlo_integrate(log_measure, integrand, reduced_vars):
+    # FIXME: how to pass rng_key to here?
     sample = log_measure.sample(reduced_vars, monte_carlo.sample_inputs)
```

Comment:
The `integrate` test with monte_carlo interpretation at `test/test_integrate.py` is failing due to this issue.